server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -266,7 +266,8 @@ private synchronized void startLogMaker() {
         DfsLogger alog = null;
 
         try {
-          alog = new DfsLogger(tserver.getContext(), syncCounter, flushCounter);
+          alog = DfsLogger.fromCounters(tserver.getContext(), syncCounter, flushCounter,
+              tserver.getClientAddressString());
           alog.open(tserver.getClientAddressString());
```

Comment:
This is the spot referenced in the ticket and I think can be addressed by my other comments.